### PR TITLE
fix: #736 parse ramq before sending fix prescription display

### PR DESCRIPTION
**Pull request by francisprovost**

### CODE REVIEW

File: client/src/components/screens/PatientSubmission/components/ClinicalInformation/components/MrnItem.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ interface Props {
 const MrnItem: React.FC<Props> = ({ form, onChange }) => {
   const [mode, setMode] = useState<Mode>(Mode.SELECT);
 
-  const patient = useSelector<State>((state) => state.patient.patient.original) as Patient;
+  const patient = useSelector<State>((state) => state.patientSubmission.patient) as Patient;
```
Comment (YanLabCHU) [inline client/src/components/screens/PatientSubmission/components/ClinicalInformation/components/MrnItem.tsx:29]: Why the change?

Comment (francisprovost) [reply client/src/components/screens/PatientSubmission/components/ClinicalInformation/components/MrnItem.tsx:29]: When you just created a patient `patient` is empty. And we're in the form so we should use the "right" data

**Review by YanLabCHU — APPROVED** [review]
🚀
